### ISSUE TITLE
Is there a way to make a fake satellite signal for satellaview bios?

### ISSUE THREAD

**Issue by TauaRian** [snes9xgit/snes9x#859]
Is there a way to make a fake satellite signal for satellaview bios?

**Comment by bearoso** [comment]
Yes, you could do this. But the offerings were entirely bulk downloads like games, so it's much easier to just have a dump of the downloads already integrated in a ROM package. I understand there's a novelty about using the original interface, but not a lot of practicality in implementing it.

LuigiBlood is the one with the most knowledge of this protocol and has a project like that you can look into: https://project.satellaview.org
